spec/README.md

```diff
@@ -15,6 +15,10 @@ It has four layers that are independent but designed to work together:
 -   [Envelope]: Handles authentication and serialization.
 -   [Bundle]: Defines a method of grouping multiple attestations together.
 
+The following diagram visualises the relationships between the envelope, statement and predicate layers.
+
+![Relationships between the envelope, statement and predicate layers](../images/specification.png)
```

Comment:
nit: could you please make the image size smaller? It's currently cut off about half-way on smaller screens (even on my 13 in laptop).

Reply:
Thanks. Done. :)